src/providers/autocomplete/AutocompleteSources.cpp

```diff
@@ -0,0 +1,430 @@
+#include "providers/autocomplete/AutocompleteSources.hpp"
```

Comment:
Make a new directory for the CompletionSources and split each up into their own file

so we'd have:
 - `src/controllers/completion/sources/unified.hpp` + its cpp file
 - `src/controllers/completion/sources/commands.hpp`
 - `src/controllers/completion/sources/users.hpp`
 - etc...